### PR TITLE
AC Test fix - LRAC-10771 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10771

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by joshitagaki — APPROVED** [review]
Nice LGTM! Yea I guess csv isn't necessary for these tests

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase [hunk 1]
```diff
@@ -91,42 +87,34 @@ definition {
...
-		task ("Add a CSV Data Source") {
-			ACSettings.addCSVDataSource(
-				dataSourceName = "CSV Data Source",
-				sourceFileName = "jobTitle.csv");
-		}
-
-		task ("Exit Settings") {
-			ACNavigation.exitSettings();
-		}
-
```
Comment (christianstokes) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase:103]: Looking at this test before, I am confused why it ever added a CSV file in the first place 🤔

**Review by christianstokes — APPROVED** [review]
Looks like a good fix to me. The CSV is confusing in this test, I dont know why it was ever there. Approved.

**Comment by christianstokes** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase (hunk 1 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase:103]: I was confused too. I think he thought the graph only worked with CSV. This test was from the automation task force, not everyone knows how things worked.

**Pull request closed without merging**